x-pack/plugins/ml/public/application/data_frame_analytics/pages/job_map/job_map.tsx

```diff
@@ -37,102 +37,78 @@ ${theme.euiColorLightShade}`,
   marginTop: 0,
 };
 
-export const JobMapTitle: React.FC<{ analyticsId?: string; modelId?: string }> = ({
-  analyticsId,
-  modelId,
-}) => (
-  <EuiTitle size="xs">
-    <span>
-      {analyticsId
-        ? i18n.translate('xpack.ml.dataframe.analyticsMap.analyticsIdTitle', {
-            defaultMessage: 'Map for analytics ID {analyticsId}',
-            values: { analyticsId },
-          })
-        : i18n.translate('xpack.ml.dataframe.analyticsMap.modelIdTitle', {
-            defaultMessage: 'Map for trained model ID {modelId}',
-            values: { modelId },
-          })}
-    </span>
-  </EuiTitle>
-);
-
-interface GetDataObjectParameter {
-  id: string;
-  type: string;
-}
-
 interface Props {
   analyticsId?: string;
   modelId?: string;
 }
 
 export const JobMap: FC<Props> = ({ analyticsId, modelId }) => {
-  const [elements, setElements] = useState<cytoscape.ElementDefinition[]>([]);
-  const [nodeDetails, setNodeDetails] = useState({});
-  const [error, setError] = useState(undefined);
+  const [itemsDeleted, setItemsDeleted] = useState<boolean>(false);
+  const [resetCy, setResetCy] = useState<boolean>(false);
```

Comment:
The data "flow" of `resetCy` wasn't obvious to me first. The name and type implies that it will reset when `true`, but the action actually toggles the value between `true` and `false` and every toggle will trigger the reset action. The code itself is fine, but maybe it's worth adding a comment with a description here how it works.

Reply:
I added a comment for clarity 626ab62bb02d68546b25cf3cbf527a0fd88cc1a9